ReactViewControl/ReactViewRender.cs

```diff
@@ -588,13 +595,22 @@ internal partial class ReactViewRender : IChildViewHost, IDisposable {
             }
         }
 
-        private FrameInfo GetOrCreateFrame(string frameName) {
-            if (!Frames.TryGetValue(frameName, out var frame)) {
-                frame = new FrameInfo(frameName);
-                Frames[frameName] = frame;
-                AddPlugins(PluginsFactory(), frame);
-            }
-            return frame;
+        private FrameInfo GetOrCreateFrame(string frameName) {
+            if (Frames.TryGetValue(frameName, out var frame)) {
+                return frame;
+            }
+
+            if (RecoverableFrames.TryGetValue(frameName, out var weakReferenceFrame) && weakReferenceFrame.TryGetTarget(out var recoverableFrame)) {
+                Frames[frameName] = recoverableFrame;
+                RecoverableFrames.Remove(frameName);

```

Comment:
Just a question. As we are adding all Frames, is it possible that we end accumulating frames on RecoverableFrames that are never recreated?

Reply:
Update: Problem solved with suggestions from Neves that saw only now

Reply:
theres a known issue.. we might end up with garbage (entries) on the RecoverableFrames dictionary without value ()once the weak ref goes away)... only the strings (key). We could have a method to periodically clear the dictionary... although the gains might be residual (we're just storing strings).